addons/account_payment/models/account_move.py

```diff
@@ -102,12 +102,12 @@ def action_view_payment_transactions(self):
 
     def _get_default_payment_link_values(self):
         next_payment_values = self._get_invoice_next_payment_values()
-        amount_max = next_payment_values['amount_due']
+        amount_max = next_payment_values.get('amount_due')
         additional_info = {}
         open_installments = []
-        if next_payment_values['installment_state'] in ('next', 'overdue'):
+        if next_payment_values.get('installment_state') in ('next', 'overdue'):
```

Comment:
Maybe we can have a separate variable for `installment_state` and `next_amount_to_pay`,

Reply:
> Maybe we can have a separate variable for `installment_state` and `next_amount_to_pay`,

The code has been updated according to your suggestions. Thank you :)